testing_dir/minesweeper.py

```diff
@@ -89,7 +89,7 @@ def game_loop():
             player_pos = new_pos
             moves += 1
 
-        if moves == BOARD_SIZE * BOARD_SIZE - NUM_MINES:
+        if moves == BOARD_SIZE * BOARD_SIZE - NUM_MINES + 1:  # Error: should not have +1
```

Comment:
The condition for checking if the game is over is incorrect as it improperly includes `+1`. The correct condition should be without the off-by-one error.
```diff
-        if moves == BOARD_SIZE * BOARD_SIZE - NUM_MINES + 1:
+        if moves == BOARD_SIZE * BOARD_SIZE - NUM_MINES:
```